Website/src/rendering/src/assets/css/components/PageContent/ThreeColumnsCta.css

```diff
@@ -10,7 +10,7 @@
       @apply lg:py-0 md:px-8;
 
       .cta-heading {
-        @apply text-blue-lightest text-5xl font-bold;
+        @apply text-blue-lighter text-5xl font-bold;
```

Comment:
This is not related to the current work item. Where is that change requirement is coming from?

Reply:
added a lighter blue than existing
so renamed lightest to lighter 
and created lightest to be used in welcome message bg